### ISSUE TITLE
Helptext not reachable with screen reader

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#817]
### Description of the bug

You cant read content in a helptext (tooltip) after it has been expanded. This should maybe not read automaticly, but you should be able to reach it via keyboard navigation. 

This is tested with both NVDA and VoiceOver

### Steps To Reproduce

1. Go to an app with a help text
2. Use a screen reader and navigate the app using your keyboard
3. Try to get the screen reader to read the text within the tooltip.
When it is open it should be within reach of the screen reader

### Additional Information

- [ ] Design task: Research what went wrong and how other systems handle this problem

### Sources
[Solution suggestion](https://www.accessibility-developer-guide.com/examples/widgets/tooltips/#manually-displayed-tooltip)
[Navs helptext component](https://aksel.nav.no/komponenter/core/helptext)

**Comment by Magnusrm** [comment]
Should be fixed by new helptext-component: https://github.com/Altinn/altinn-design-system/issues/253
